Edting pr guidelines (#5)
mdn/temp-processes: pull-requests.md

# Pull request submission and review guidelines

This document describes how contributors make changes to MDN Web Docs and how the changes are reviewed and land on the site.

> We like pull requests that close existing issues and are part of bigger project work. You can find tasks to help with on repo issues trackers and our public project boards. Please make sure before embarking on any work that you're not making changes that are already being made. Search issue trackers and open prs.
> If you are looking to suggest a new feature or a new set of work please submit a proposal through the [suggestion form]().

Reviewer: ```suggestion

We strongly recommend opening pull requests that close existing issues or that are part of a bigger project. You can find tasks to help with under 'Issues' in any of the [MDN repositories](https://github.com/orgs/mdn/repositories) (for example, [Issues](https://github.com/mdn/content/issues) in `mdn/content` repository) and our public [project boards](https://github.com/orgs/mdn/projects).

Before embarking on any work in any of the MDN repositories, please make sure that you're not making changes that are already being made. You can do this by searching through the open 'Issues' and open 'Pull requests' in the repository you're interested in making changes.

If you are looking to suggest a new feature or a new set of work, please submit a proposal through the [suggestion form]().
```

If anyone has engaged in behavior that is potentially illegal or makes you or someone else feel unsafe, unwelcome, or uncomfortable, you are encouraged to [report it](https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/).

## Opening a pull request

### If you find something you want to work on

Ideally you'll have found an issue or project task to work on before opening a pull request. Every pull request opened should be to close an existing issue. If you haven't check through issues on repos or public project boards to find something. If you have found a problem on MDN open an issue - more [information on issues can be found here](). **Issues need to be triaged before work can start.**

Make sure the issue isn't assigned and no one has already opened a pull request for the task. You can do that by checking the issue and searching open pull requests

If you're not sure where to start, reach out to us on [Discord]() and ask for feedback before starting on any work.

### Opening a pull request

When you're ready to open a pull request, follow these guidelines:

> - **PRs should be short and focused to one issue** or a related set of changes.
>   If a PR becomes too large, the reviewer may close it and ask that you submit pull requests for each logical set of changes that belong together.

Reviewer: ```suggestion
- **PRs should be short and focused to one issue:** If possible, group related set of changes into multiple, small PRs. If a PR becomes too large, the reviewer may close it and ask that you to submit pull requests for each logical set of changes that belong together.
```

> - **Add a description of what the pull request is doing**

Reviewer: ```suggestion
- **Add a description of what the pull request is doing:** Provide as much context and reason for opening the pull request.
```

> - **Add the link to the issue you are closing or working on** Add 'Fixes' or 'Relates to' and the related issue to the description of the pull request. See [github link here]()
> - **Accompany code example changes with content changes** to ensure updated examples are served correctly.
>   If content changes also affect how examples are used, the related code examples should be updated in associated repositories.

Reviewer: ```suggestion
- **Add the link to the issue you are closing or working on:** Add 'Fixes' or 'Relates to' and the related issue in the description of the pull request. See [github link here]()
- **Accompany code example changes with content changes:** This is important to ensure that updated examples are served correctly. If content changes also affect how examples are used, the related code examples should be updated in associated repositories.
```

## Pull request review process

We add reviewers automatically based on a `CODEOWNERS` file but, if there is a specific person you want to request review from, manually [request a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review).
We also use auto-labeling on pull requests to help triage.

> Maintainers will triage pull requests and add any additional labels if needed based on context. These are usually as and when are needed like `needs-info` or `on-hold`.

Reviewer: ```suggestion
Maintainers will triage pull requests and add any additional labels, such as `needs-info` or `on-hold`, if needed based on context.
```
